pc_autobackup.py

```diff
@@ -163,8 +176,6 @@ def main():
   parser.add_option('-b', '--bind', dest='bind',
                     help='bind the server to a specific IP',
                     metavar='IP')
-  parser.add_option('--config_file', dest='config_file',
```

Comment:
Please revert all of this or make it work using the --config_file option. You are forcing users to use a hardcoded config file instead of allowing them to choose which config file they'd like to use. Not only is that not great, but you will break all users (including me) who are using config files outside of the running users homedir. I have my config file placed in /etc/pc_autobackup.conf and upgrading to this version will break my daemon.